executor/platform_compat/windows/pty_manager.py

```
@@ -0,0 +1,192 @@
+            return b""
+        except Exception:
+            return b""
+
+    def write(self, data: bytes) -> int:
+        """Write data to the PTY."""
+        try:
+            if isinstance(data, bytes):
+                data = data.decode("utf-8", errors="replace")
+            self._pty.write(data)
+            return len(data)
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)